XCUITests/DatabaseFixtureTest.swift

```diff
@@ -24,8 +24,11 @@ class DatabaseFixtureTest: BaseTestCase {
 
     func testHistoryDatabaseFixture() {
         navigator.goto(HomePanel_History)
+        // Small delay to avoid intermittent failures
+        // Ideally, we would wait for the table to become populated, not sure how to do that.
+        sleep(1)
```

Comment:
I saw intermittent failures locally without adding this.

Reply:
Can we try with ```waitforExistence("app.tables["History List"]")``` ? instead of the sleep. If that does not work to avoid the intermittents then lets use the sleep but like the last resource

Reply:
We need to wait until the history list has rows, not for it to exist. The rows are created async.